lib/features/home/view_model/get_properties_state.dart

```diff
@@ -0,0 +1,44 @@
+part of 'get_properties_cubit.dart';
+
+abstract class GetPropertiesState extends Equatable {
+  const GetPropertiesState();
+
+  bool get isInitialState => this is GetPropertiesInitialState;
+  bool get isGettingPropertiesState => this is GettingPropertiesState;
+  bool get isGetPropertiesSucceed => this is GetPropertiesSucceedState;
+  bool get isGetPropertiesFailed => this is GetPropertiesFailedState;
+
+  List<PropertyItemData> get properties =>
+      (this as GetPropertiesSucceedState).propertyList;
+
+  String get errorFetching => (this as GetPropertiesFailedState).error;
```

Comment:
Move this into GerPropertyFailedState

Reply:
@suesitran I moved data and error to parent class because i don't want to check state class type in UI code. 
About moving get properties from parent class to GetPropertiesSucceedState class and errorFetching to GetPropertiesFailedState, i don't think it make sense, because if anyone plans to use GetPropertiesSucceedState or GetPropertiesFailedState for checking state, they can directly get data/error from those class without using getter, and casting.